sssd_test_framework/utils/sbus.py

```diff
@@ -0,0 +1,476 @@
+"""Manage D-Bus operations"""
+
+from __future__ import annotations
+
+import xml.etree.ElementTree as xml
+from abc import ABC
+from typing import Any, Final
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.ssh import SSHProcessResult
+
+from sssd_test_framework.utils.dbus_types import (
+    DBUSResult,
+    DBUSSignatureReader,
+    DBUSType,
+    DBUSTypeString,
+    DBUSTypeVariant,
+)
+
+__all__ = [
+    "SBUSUtils",
+    "DBUSKnownBus",
+]
+
+
+class DBUSKnownBus(str):
+    """
+    Weel-known D-Bus buses.
+    """
+
+    SYSTEM: Final = "--SYSTEM--"
+    """The system bus"""
+
+    SESSION: Final = "--SESSION--"
+    """The session bus"""
+
+    MONITOR: Final = "unix:path=/var/lib/sss/pipes/private/sbus-monitor"
+    """Monitor's private bus"""
+
+    @classmethod
+    def forDomain(cls, name: str):
+        """Produce the bus for the given domain"""
+        return cls(f"unix:path=/var/lib/sss/pipes/private/sbus-dp_{name}")
```

Comment:
It's tempting to add this here, but this is for internal communication with SSSD, so not something we should be testing in system tests. The system tests are black box tests, we should not expect any implementation details. We can only tests stuff that is visible to the user (only the infopipe currently).